UnityAds/UnityAdsRewardedVideoCustomEvent.m

```diff
@@ -85,21 +86,20 @@ - (void)requestAdWithAdapterInfo:(NSDictionary *)info adMarkup:(NSString *)adMar
         [[UnityRouter sharedRouter] initializeWithGameId:gameId withCompletionHandler:nil];
         
         NSError *error = [NSError errorWithDomain:MoPubRewardedVideoAdsSDKDomain code:MPRewardedVideoAdErrorInvalidCustomEvent userInfo:@{NSLocalizedDescriptionKey: @"Unity Ads adapter failed to request rewarded video ad. Unity Ads is not initialized yet. Failing this ad request and calling Unity Ads initialization so it would be available for an upcoming ad request.", NSLocalizedRecoverySuggestionErrorKey: @""}];
-        MPLogAdEvent([MPLogEvent adLoadFailedForAdapter:NSStringFromClass(self.class) error:error], [self getAdNetworkId]);
+        MPLogAdEvent([MPLogEvent adLoadFailedForAdapter:NSStringFromClass(self.class) error:error], placementID);
         [self.delegate fullscreenAdAdapter:self didFailToLoadAdWithError:error];
         return;
     }
     
-    MPLogAdEvent([MPLogEvent adLoadAttemptForAdapter:NSStringFromClass(self.class) dspCreativeId:nil dspName:nil], [self getAdNetworkId]);
+    MPLogAdEvent([MPLogEvent adLoadAttemptForAdapter:NSStringFromClass(self.class) dspCreativeId:nil dspName:nil], placementID);
     [UnityAds load:self.placementId loadDelegate:self];
 }
 
 - (void)presentAdFromViewController:(UIViewController *)viewController
 {
     if ([self hasAdAvailable]) {
```

Comment:
To align with Android, should we remove this?

Ideally we can offload everything to our SDK, and will trip a failsafe from our end and correctly fail there

Reply:
I think the only issue would be, if we do remove it and for some reason the placement is not loaded before it is attempted to be shown, the show failed delegate will be called with an error of `UADSAbstractModule cannot continue with the empty placement ID` which might be a confusing error message in this context.  If we handle it here, then we can maybe provide a better error about it not having been loaded yet.

Reply:
I see - Then this is something that we currently can't support due to our native implementation. Thanks for the context!

Reply:
Also on android, we might not have the `hasAdAvailable` method, but it looks like in the updated PR we are checking if it loaded before showing: https://github.com/Applifier/googleads-mobile-android-mediation/pull/17/files#r601708202